src/acl-control.js

```diff
@@ -612,14 +612,19 @@ UI.aclControl.ACLControlBox5 = function (subject, dom, noun, kb, callback) {
           var q = str.indexOf('//')
           var targetDocDir = ((q >= 0 && p < q + 2) || p < 0) ? null : str.slice(0, p + 1)
 
-          if (targetDocDir) {
+          // @@ TODO: The methods used for targetIsStorage are HACKs - it should not be relied upon, and work is
+          // @@ underway to standardize a behavior that does not rely upon this hack
+          // eslint-disable-next-line no-undef
```

Comment:
Is the `no-undef` disabling due to `UI`? Might be better to tell eslint that it's an available global, then.

(And if it's about `location`, perhaps best to use `window.location`.)